src/core/multifeaturelistmodel.h

```diff
@@ -72,6 +73,13 @@ class MultiFeatureListModel : public QSortFilterProxyModel
      */
     Q_INVOKABLE void clearSelection();
 
+    /**
+     * Returns the display name of a given feature.
+     * \param vlayer the vector layer containing the feature
+     * \param feature the display name feature
+     */
+    Q_INVOKABLE QString featureDisplayName( QgsVectorLayer *vlayer, QgsFeature feature );
```

Comment:
```suggestion
    Q_INVOKABLE QString featureDisplayName( QgsVectorLayer *vlayer, const QgsFeature &feature ) const;
```

... and evenmore: does this belong in here -- or rather into a singleton/static "utils" class?